github/tasks.rake

```diff
@@ -1,4 +1,6 @@
-require_relative 'lib/configure_repos'
+require_relative "lib/configure_repos"
+require_relative "lib/validate_repos"
+require_relative "lib/fetch_repos"
```

Comment:
We can remove the `fetch_repos` line, as we only call it indirectly (via ConfigureRepos and ValidateRepos, and these already require the file themselves)

Reply:
`fetch_repos` requirement has now been removed from rakefile.